### PR TITLE
Document linked devices

### PR DESCRIPTION
**Pull request by vlindhol**
Note that this depends on PR #122.

 ### Summary

There was not much prose about the device details page, so I
added/reordered some headings and wrote a bit more than just
about linked devices:

* Create a new page `Projects and Fleets`
  * Fold `Create a new Project` page into there
  * Document Cohorts there
  * Move Device Set docs there
* Create a new page `Inspecting a Device`
  * Briefly describe the device details page there
  * Document linked devices there

### CODE REVIEW

File: docs/platform/projects-and-fleets.mdx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+unit or a games console. Fleets may also consist of different but related
+devices, for example left and right wireless earbuds. We recommend keeping
+different revisions of the same hardware in the same Project, so that revisions
+can be easily compared later.
+
+Within each Memfault Project you can:
+
+- [Monitor Fleet-wide metrics](./fleet-wide-metrics.mdx)
+- [Inspect and debug individual Devices](./inspecting-a-device.mdx)
+- Deploy new firmware/software updates to parts of your Fleet
```
Comment (martijnthe) [inline docs/platform/projects-and-fleets.mdx:24]: Add links to the "Release Management" pages under MCU Guides / Android Guides?

File: docs/platform/projects-and-fleets.mdx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+It is often useful to search for devices that fulfill certain criteria and to
+keep track of the number of matches over time. Examples of this could be
+searching for devices that:
+
+1. have not communicated with Memfault within the last N days,
+2. are running a deprecated software version, or
+3. are reporting a metric over a critical threshold
+
+This kind of followup can easily be done by defining a **Device Set**, which
```
Comment (martijnthe) [inline docs/platform/projects-and-fleets.mdx:71]:
```suggestion
This kind of follow-up can easily be done by defining a **Device Set**, which
```

**Comment by HBehrens** [comment]
I read in the comment that this one depends on #122. How do we manage this dependency though considering that we didn't plan to work on Device Sets until after Linking was done?

File: docs/platform/inspecting-a-device.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+:::
+
+In many situations a Device does not operate in a vacuum. When debugging e.g.
+wireless earbuds, the left and right bud is intimately linked, and misbehavior
+in one can cause a crash in the other. Memfault enables your Devices to report
+which other Devices they are in some way linked to, whatever "linked" means in
+your particular case.
+
+![Animation showing the mouse hovering over a linked Device tag in the Device details page](/img/docs/platform/linked-devices-hover.gif)
```
Comment (martijnthe) [inline docs/platform/inspecting-a-device.mdx:37]: The screenshots are no longer up to date, sorry!

**Pull request merged**